Sources/TSCBasic/FileSystem.swift

```diff
@@ -179,11 +185,16 @@ public protocol FileSystem: class {
     /// Change file mode.
     func chmod(_ mode: FileMode, path: AbsolutePath, options: Set<FileMode.Option>) throws
 
-
     /// Returns the file info of the given path.
     ///
     /// The method throws if the underlying stat call fails.
     func getFileInfo(_ path: AbsolutePath) throws -> FileInfo
+
+    /// Copy a file or directory.
+    func copyItem(at sourcePath: AbsolutePath, to destinationPath: AbsolutePath) throws
```

Comment:
It looks as if we're currently not using the term "item", so maybe we should simplify these to `copy()` and `move()`?